client/src/components/Home/Chat.js

```diff
@@ -0,0 +1,230 @@
+import React, { Component } from 'react'
```

Comment:
Was this file just moved/renamed? I can't find the deleted portion in this PR.

Reply:
File added by mistake. Removed.